src/main/resources/templates/db_scripts/softdelete/softDeleteForHoldingsInstancesItems.sql

```diff
@@ -0,0 +1,20 @@
+-- add deleted_date field to the tables
+-- create indexees on deleted date
+-- rename instance, holdings and item tables
+-- create views with the name of the corresponding tables
+
+ALTER TABLE ${myuniversity}_${mymodule}.instance ADD "deleted_date" TIMESTAMP;
+ALTER TABLE ${myuniversity}_${mymodule}.item ADD "deleted_date" TIMESTAMP;
+ALTER TABLE ${myuniversity}_${mymodule}.holdings_record ADD "deleted_date" TIMESTAMP;
+--
+CREATE INDEX instance_deleted_date__index ON ${myuniversity}_${mymodule}.instance (deleted_date);
+CREATE INDEX item_deleted_date__index ON ${myuniversity}_${mymodule}.item (deleted_date);
+CREATE INDEX holdings_record_deleted_date__index ON ${myuniversity}_${mymodule}.holdings_record (deleted_date);
+--
+ALTER TABLE ${myuniversity}_${mymodule}.instance RENAME TO instance_log;
```

Comment:
It seems to me that after renaming the table we won't be able to add/remove an index/FK/etc to these tables via RMB standard capabilities (because table name in schema.json does not match actual table name).